styleguide/derek/global-components/tracking/tracking.spec.js

```diff
@@ -228,6 +228,81 @@ describe('Zoolander Tracking Module', () => {
       expect(expected).to.eql(window.dataLayer.pop());
     });
 
+    it('should track internal nav clicks', () => {
+      const linkHtml = '<div class="navbar-menuContainer" id="main-navigation"><ul class="top-nav">' +
+          '<li class="item-i"><span class="navbar-topLink">Top lvl first</span></li> ' +
+          '<li class="item-ii"><span class="navbar-topLink">Top lvl second</span>' +
+            '<ul class="mid-nav">' +
+              '<li class="item-a"><a class="navbar-dropDownLink">Mid lvl first</a></li>' +
+              '<li class="item-b"><span id="midlvl">Mid lvl second</span>' +
+                '<ul class="third-nav">' +
+                  '<li class="item-1">Base lvl first</li>' +
+                  '<li class="item-2"><a class="navbar-tertiary-dropDownLink">Base lvl second</a></li>' +
+                  '<li class="item-3">Base lvl third</li>' +
+                '</ul>' +
+              '</li>' +
+              '<li class="item-c">Mid lvl third</li>' +
+            '</ul>' +
+          '</li>' +
+          '<li class="item-iii"><a class="navbar-topLink">Top lvl third</a></li>' +
+        '</ul>' +
+      '</div>';
+
+      $('body').append(linkHtml);
+      Zoolander.Tracking.init();
+
+      // click base level
+      $('.item-2 .navbar-tertiary-dropDownLink').trigger('click');
+      let expected = {
+        event: 'ga.event',
+        eventCategory: 'Internal Links',
+        eventAction: 'Link Click - Navigation',
+        eventLabel: 'Base lvl second:Mid lvl second:Top lvl second',
```

Comment:
@xinbin I read the acceptance criteria and it shows the label being in reverse order than what is shown here. Am I seeing things wrong?